ciscripts/buildpending.sh

```diff
@@ -2,5 +2,6 @@
 curl -XPOST -H "Authorization: token $GITHUB_SECRET" https://api.github.com/repos/steemit/steem/statuses/$(git rev-parse HEAD) -d "{
```

Comment:
@mvandeberg - It needs to be approved and rebased with all branches or you'll see that 'UI error' - jenkins was reporting it's status as 'default' and I updated the context it reports in JSON to github as part of this pull request.